lib/cra-template/package.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
I think there's an outstanding issue where the `main` field is required by CRA.

https://github.com/facebook/create-react-app/pull/8560

We plan to fix it, but for now it might be safer to add that field. In the CRA templates we set to `template.json`.

Reply:
Good to know! Thx!